toolkit/livedata/README.md

```diff
@@ -0,0 +1,32 @@
+[ ![Download](https://api.bintray.com/packages/methe/arch-toolkit/livedata/images/download.svg) ](https://bintray.com/methe/arch-toolkit/livedata/_latestVersion)
+[![CircleCI](https://circleci.com/gh/matheus-corregiari/arch-toolkit/tree/master.svg?style=svg)](https://circleci.com/gh/matheus-corregiari/arch-toolkit/tree/master)
+
+# Livedata
+
+Some custom implementations of LiveData and some extensions
+
+### Usage
+
+#### Add into your project
+
+###### build.gradle
+
+First you need to add the Google Architecture components
+
+```groovy
+implementation "android.arch.lifecycle:livedata:$versions.arch"
+```
+
+Then add the Livedata
```

Comment:
It would be better if this instruction was: `Then add the Livedata implementation from arch-toolkit`